api/src/api/v1/prerequisites/require_child_organisation.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * "organisations_details-child" route pre-requisite
+ *
+ * Determines if the authenticated user is trying to access an organisation
+ * which is actually a "child" entity.
+ *
+ * Conditions under which this is true for organisation X:
+ * - User is an admin for a funding body which owns at least
```

Comment:
"funding body" concept no longer exists. I think we shouldn't include this in our descriptions anymore